cradle-core/src/main/java/com/exactpro/cradle/CradleStorage.java

```diff
@@ -1461,7 +1461,22 @@ public PageInfo updatePageName(BookId bookId, String pageName, String newPageNam
         PageInfo updatedPageInfo = doUpdatePageName(bookId, pageName, newPageName);
 
         try {
-            updatePage(new PageId(bookId, pageName), updatedPageInfo);
+            // FIXME: implement search in data base
+            updatePage(new PageId(bookId, null, pageName), updatedPageInfo);
+        } catch (Exception e) {
+            logger.error("Page was edited but cache wasn't refreshed, try to refresh pages");
+            throw e;
+        }
+
+        return updatedPageInfo;
+    }
+
+    public PageInfo updatePageName(BookId bookId, Instant pageStart, String pageName, String newPageName) throws CradleStorageException {
+        getBookCache().getBook(bookId);
```

Comment:
Why do we call this here and don't use the result? If this is done for some reason we should probably leave a comment here with that reason